src/main/java/com/example/ajaxonboardingproject/security/jwt/JwtSecurityContextRepository.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.ajaxonboardingproject.security.jwt
+
+import org.springframework.stereotype.Component
+import org.springframework.web.server.ServerWebExchange
+import reactor.core.publisher.Mono
+import org.springframework.http.server.reactive.ServerHttpRequest
+import org.springframework.security.core.context.SecurityContext
+import org.springframework.security.core.context.SecurityContextImpl
+import org.springframework.security.web.server.context.ServerSecurityContextRepository
+
+@Component
+class JwtSecurityContextRepository(
+    private val jwtTokenProvider: JwtTokenProvider
+) : ServerSecurityContextRepository {
+
+    override fun save(swe: ServerWebExchange, sc: SecurityContext): Mono<Void> {

```

Comment:
```suggestion
    override fun save(swe: ServerWebExchange, sc: SecurityContext): Mono<Unit> {

```

Let's use kotlin's "void"

Reply:
I can not use it because i override method with return type Mono<Void>